configmap/hash-gen/main.go

```diff
@@ -0,0 +1,125 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"gopkg.in/yaml.v3"
+	"knative.dev/pkg/configmap"
+)
+
+func main() {
+	fileName := os.Args[1]
+	if err := processFile(fileName); err != nil {
+		log.Fatal(err)
+	}
+}
+
+func processFile(fileName string) error {
```

Comment:
A small Godoc explaining what "process" means in this context could help future readers. E.g. 
> processFile reads the ConfigMap manifest from a file and adds or updates the label containing the hash of its _example data.